src/worker-bindings.cc

```diff
@@ -154,6 +155,48 @@ const char* Worker::InitIsolate() {
   return NULL;
 }
 
+void Worker::WriteCodeCoverageReport() {
+  Locker locker(isolate_);
+  Isolate::Scope isolate_scope(isolate_);
+  Nan::HandleScope scope;
+
+#if NODE_VERSION_AT_LEAST(0, 11, 0)
+  Local<Context> context = Local<Context>::New(isolate_, context_);
+  Context::Scope context_scope(context);
+#else
+  Handle<Context> context = context_;
+  Context::Scope context_scope(context_);
+#endif
```

Comment:
I'd pass in `isolate_` and `context_` explicitly and have a `context` property or getter.  Only go through the trouble if there are at least three places where this is duplicated, else it's not worth the effort.